notebooks/sparse/quickstart.ipynb

```diff
@@ -37,15 +36,12 @@
         "# Install the required packages.\n",
         "\n",
         "import os\n",
+        "!pip install torch==2.0.0+cpu torchvision==0.15.1+cpu torchaudio==2.0.1 --index-url https://download.pytorch.org/whl/cpu > /dev/null\n",
         "import torch\n",
         "os.environ['TORCH'] = torch.__version__\n",
         "os.environ['DGLBACKEND'] = \"pytorch\"\n",
         "\n",
-        "# Uncomment below to install required packages. If the CUDA version is not 11.8,\n",
```

Comment:
why remove?

Reply:
It is not trivial for a new user to figure the CUDA version used in colab. So I changed the default runtime to CPU, which can also effectively demonstrate our sparse library.

Reply:
Let's still leave `"# Uncomment below to install required packages.`.

Reply:
Please update the other examples in a separate PR as well.

Reply:
I feel it is better to specify both Pytorch and DGL version in the tutorials because there is always a period of time that the sparse library is not compatible with the latest Pytorch version.

Reply:
We should comment out the install command to avoid rush in the doc builder.

Reply:
> Please update the other examples in a separate PR as well.

OK.

> We should comment out the install command to avoid rush in the doc builder.

Done.

Reply:
Please keep an eye on https://docs.dgl.ai/notebooks/sparse/quickstart.html after PR merge.